Homeworks/HW_5/myproject/myapp5/admin.py

```diff
@@ -0,0 +1,120 @@
+from django.contrib import admin
+from .models import User, Product, Order
+
+
+@admin.action(description="Обнулить количество")
+def reset_quantity(modeladmin, request, queryset):
+    queryset.update(quantity=0)
+
+
```

Comment:
Для регистрации кастомных моделей можно использовать декораторы 
```suggestion
@admin.register(User)
```